cpp/include/cudf/fixed_point/temporary.hpp

```diff
@@ -15,6 +15,9 @@
  */
 
 #pragma once
+// To avoid https://github.com/NVIDIA/libcudacxx/issues/460
+// in libcudacxx with CTK 12.0/12.1
+#include <cuda_runtime.h>
```

Comment:
It looks like they made a fix upstream ( https://github.com/NVIDIA/libcudacxx/pull/476 ). Is it possible for us to get that fix and drop this workaround?

Reply:
This fix hasn't been released yet. We're currently pinned to libcudacxx 1.9.1, which is decently far behind the latest version of 2.1.0. I might consider patching this if we were already on 2.1.0, but we're not close enough to the HEAD to apply this patch yet. Note that upgrading to CCCL 2.1.0 is a significant endeavor that will affect all of RAPIDS, and #13222 is the PR where I'm working on that.